test/fixtures/globala.js

```diff
@@ -0,0 +1,5 @@
+global.a = 'test';
+process.stdin.on('data', function(chunk) {
```

Comment:
Strange, it wasn't working for me. The process never ended until I added this. I'll take another look though